src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -58,6 +57,10 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
             }
         }
 
+        associatedCaseLinkHelper.linkCaseByNino(sscsCaseData);
```

Comment:
Be good to capture the returned value and use that for returning rather than rely on pass by reference.

Reply:
updated